packages/nimble-components/src/table-column/number-text/tests/table-column-number-text.mdx

```diff
@@ -20,11 +20,16 @@ based on the value of the `lang` token, which can be set via the [nimble-theme-p
 
 {/* ## Appearance Variants */}
 
-{/* ## Usage */}
+## Usage
 
-## Angular Usage
+### Best Practices
 
-### Number Format Pipe
+-   Avoid mixing `undefined` and `null` as values for the same field. When grouping this will lead to two groups (one for `null` values and one for `undefined` values) that both have the text `"No value"`.
+-   If relevant to your data source, make sure to consider the IEEE 754 special cases of `-Inf`, `+Inf`, `-0`, `+0`, and `NaN`.
```

Comment:
What is the guidance/best practice here? A client might infer that the column doesn't handle those values. What about these values do they need to consider?